Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1125,6 +1120,8 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, two_be
     if 'NonSupported' in packs_to_install:
         packs_to_install.remove("NonSupported")
 
+    packs_to_install.add("DeveloperTools")
```

Comment:
@dantavori are you sure about it? In such case, I don't think that changes in your second PR https://github.com/demisto/content/pull/7468 are relevant.